<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>751</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="live">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>601</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Live</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="live_feed_label">
       <property name="text">
        <string>live_feed_label</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="detection_review">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Detection Review</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLabel" name="detection_image">
       <property name="text">
        <string>detection_image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="response_commands">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>601</width>
      <height>106</height>
     </rect>
    </property>
    <property name="title">
     <string>Response Commands</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_case_closed">
       <property name="text">
        <string>Case Closed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_emergency_warning">
       <property name="text">
        <string>Emergency Warning</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_danger_warning">
       <property name="text">
        <string>Danger Warning</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_fire_report">
       <property name="text">
        <string>119 Report</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_police_report">
       <property name="text">
        <string>112 Report</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_illegal_warning">
       <property name="text">
        <string>Illegal Warning</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>551</width>
     <height>751</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="connectivity_label">
       <property name="text">
        <string>frame_id</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="detections">
       <property name="text">
        <string>detections</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="robot_status">
       <property name="text">
        <string>robot_status</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="map">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>531</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Map</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QLabel" name="map_display_label">
       <property name="text">
        <string>map_display_label</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="move_commands">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>531</width>
      <height>106</height>
     </rect>
    </property>
    <property name="title">
     <string>Move Commands</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_return_to_home">
       <property name="text">
        <string>Return to home</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_move_to_A">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Move to A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_move_to_B">
       <property name="text">
        <string>Move to B</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_start_video_stream">
       <property name="text">
        <string>Start Video Stream</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
